spark/v3.5/build.gradle

```diff
@@ -107,6 +107,9 @@ project(":iceberg-spark:iceberg-spark-${sparkMajorVersion}_${scalaVersion}") {
     testImplementation project(path: ':iceberg-api', configuration: 'testArtifacts')
     testImplementation project(path: ':iceberg-core', configuration: 'testArtifacts')
     testImplementation project(path: ':iceberg-data', configuration: 'testArtifacts')
+
```

Comment:
I am leaving that like here to keep test loop faster, I ll change this when we add changes on our end.